jaba/src/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,30 @@
+import classes from './ForgotPassword.module.css';
```

Comment:
Can you update to use the syntax that Ricky messaged? Use styles instead of classes